includes/views/attachment-settings-hint.php

```diff
@@ -9,6 +9,6 @@
 	<p>
 		<?php /* translators: <a> tags */ ?>
 		<?php printf( wp_kses_post( __( 'It looks like you haven\'t setup any email attachments yet, check the settings under <b>%1$sDocuments > Invoice%2$s</b>', 'woocommerce-pdf-invoices-packing-slips' ) ), '<a href="'.$invoice_settings_url.'">', '</a>' ); ?>
-		<?php printf( '<a href="%s" style="display:block; margin-top:10px;">%s</a>', esc_url( add_query_arg( 'wpo_wcpdf_hide_attachments_hint', 'true' ) ), esc_html__( 'Hide this message', 'woocommerce-pdf-invoices-packing-slips' ) ); ?>
+		<?php printf( '<a href="%s" style="display:block; margin-top:10px;">%s</a>', esc_url( wp_nonce_url( add_query_arg( 'wpo_wcpdf_hide_attachments_hint', 'true' )  ), esc_html__( 'Hide this message', 'woocommerce-pdf-invoices-packing-slips' ) ) ); ?>
```

Comment:
You have a chevron misplaced in this line

Reply:
> You have a chevron misplaced in this line

Excuse me, what does "chevron" mean?

Reply:
Do you mean the extra space I have on the line?

Reply:
Sorry, parenthese